services/terraformer/templates/hetzner.tpl

```diff
@@ -29,7 +29,7 @@ resource "hcloud_firewall" "defaultfirewall" {
     ]
   }
 
-  {{ if index .Metadata "loadBalancers" | targetPorts | isMissing 6443 }}
+  {{- if index .Metadata "loadBalancers" | targetPorts | isMissing 6443 }}
```

Comment:
> {{ - if index .Metadata ... }

Why do we need `-` before `if`?

Reply:
It's just for cleaner tf files, `-` does not result in new line creation.